mobile/lib/features/trade/trade_bottom_sheet_tab.dart

```diff
@@ -189,14 +222,16 @@ class _TradeBottomSheetTabState extends State<TradeBottomSheetTab> {
     bool hasPosition = positionChangeNotifier.positions.containsKey(contractSymbol);
 
     double? positionLeverage;
+    int usableBalance = channelTradeConstraints.maxLocalBalanceSats;
     if (hasPosition) {
       final position = context.read<PositionChangeNotifier>().positions[contractSymbol];
       positionLeverage = position!.leverage.leverage;
+      if (direction != position.direction) {
```

Comment:
🫣
```suggestion
      if (direction == position.direction.opposite())
```

Reply:
I don't think this is correct. If the user is long the open quantity can be additionally shorted.